sales/amazon_connector/new_marketplace.rst

```diff
@@ -0,0 +1,40 @@
+======================================================
+Add an unsupported marketplace to the Amazon Connector
+======================================================
+
+Some newer Amazon Marketplaces, such as Amazon Brazil or Amazon Netherlands, are
+not included by default in the Amazon Connector list of possible marketplaces.
+
+These marketplaces can be added manually should you wish to use them.
+
+.. warning::
+    These marketplaces are not officially supported by Odoo - there is no guarantee
+    that adding a new marketplace as described here will work, nor can this be considered
+    as a bug when contacting Odoo Support.
+
+.. note::
+    Amazon marketplaces are only supported in the European and North American region;
+    though Amazon includes Brazil with the North American region and India in the
+    European region, so your mileage may vary.
```

Comment:
I get the joke but not the information that comes with it ?

Reply:
If you want to add a new marketplace, it must be in the EU or NA region; but it might be in one of those even though it's not geographically speaking
e.g. you might want to add the India marketplace, since it's in europe - that's what I meant to say